CleverPush/CleverPush/Source/CPNotification.m

```diff
@@ -90,6 +90,11 @@ - (void)parseJson:(NSDictionary*)json {
         self.carouselEnabled = [[json objectForKey:@"carouselEnabled"] boolValue];
     }
 
+    self.multiscreenEnable = NO;
+    if ([json objectForKey:@"enableMultipleScreens"] && [json objectForKey:@"enableMultipleScreens"] != nil && ![[json objectForKey:@"enableMultipleScreens"] isKindOfClass:[NSNull class]]) {
```

Comment:
I think this is not needed?

Reply:
This just for handle notification flow for Banner as I have checked the code.